customer-commons/src/main/java/no/unit/nva/customer/service/CustomerService.java

```diff
@@ -23,5 +23,7 @@ public interface CustomerService {
     CustomerDto updateCustomer(UUID identifier, CustomerDto customer) throws InputException, NotFoundException;
 
     CustomerDto getCustomerByCristinId(URI cristinId) throws NotFoundException;
+    CustomerDto refreshCustomer(CustomerDto customer) throws NotFoundException;
```

Comment:
Probably not part of interface, but rather a private method in the implementation?